src/Take.Elephant/Take.Elephant.csproj

```diff
@@ -11,7 +11,7 @@
     <Copyright>Copyright © 2015 Curupira S/A</Copyright>
     <Product>Elephant</Product>
     <PackageTags>elephant</PackageTags>
-    <Version>0.5.0</Version>
+    <Version>0.10.72.2</Version>
```

Comment:
I think that we avoid commiting this change. It is useful and meaningful only for local dev.

Reply:
Truth